setup.py

```diff
@@ -88,10 +88,7 @@ def get_version(version_file):
     py_modules=[splitext(basename(path))[0] for path in glob("src/*.py")],
     include_package_data=True,
     install_requires=[
-        # Celery 5.0 introduces breaking changes to the CLI. See
-        # https://docs.celeryq.dev/en/stable/history/whatsnew-5.0.html?highlight=command%20line%20options#new-command-line-interface
-        # for more information.
-        "celery >= 4.3.0,<5.0.0",
+        "celery >= 5.1.0",
```

Comment:
Just to cover our bases. As an additional question do we *need* `5.1.0` for functionality? We should pin as low as we are able to support whenever possible.

```suggestion
        "celery >= 5.1.0,<6",
```

Reply:
I read you. To answer your question: no we don't. The lowest possible version we can support with the change is `5.0.0`, since that version introduced the breaking change. I updated my commit to reflect proper version pinning. Thanks!